packages/language-server/test/completion-items-spec.ts

```diff
@@ -138,19 +152,28 @@ describe("the UI5 language assistant Code Completion Services", () => {
     const xmlSnippet = `<mvc:View 
                           xmlns:mvc="sap.ui.core.mvc" 
                           xmlns="sap.m"> 
-                          <List update⇶Start`;
+                          <List update⇶Start="onUpdateStart"`;
```

Comment:
if we added a value should not the description of the test (scenario) also change (IT text)?
Is

Reply:
Done (I added it because this use case used to be covered by a similar test for a property attribute, which is now in a different `case` in the code)